website/public/main.css

```diff
@@ -215,11 +215,15 @@ nav.breadcrumbs ol > li:not(:first-child)::before {
   a:visited {
     color: #979ef8;
   }
+  :root {
+    color-scheme: dark;
```

Comment:
Must fix: Our website supports both dark and light modes. We shouldn't use dark components in light mode.

Based on my [reading of MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme), I think we can write `color-scheme: light dark;` to let the browser choose the right component theme. I think this should match our dark/light theme logic automatically.

Reply:
We did not use dark components in light mode.

`color-scheme: light dark;` was inside `@media (prefers-color-scheme: dark)`.

Reply:
> `color-scheme: dark;` was inside `@media (prefers-color-scheme: dark)`.

Ah, I missed this context.